### ISSUE TITLE
Command line conversion to pyramid tiff -  different behaviour between 8.9.2 (correct) and 8.10.0 - possible bug?

### ISSUE THREAD

**Issue by barryspearce** [libvips/libvips#1801]
I have the following input file:

![1500-1524c_bsip37_Vestry (formerly holy trinity chapel)_corbel_Church of St Mary,Beverley_photoFrom1986,English Bowed Instruments from Anglo-Saxon to Tudor Times by Mary Remnant,Plate144_England-001](https://user-images.githubusercontent.com/18211861/91779054-53dca600-ebec-11ea-9c46-c94fb3bf7361.jpg)

I am creating a pyramid TIFF for serving on an IIIF server. Under 8.9.2 it correctly converts. Under 8.10.0 it produces a black & white image.
Using file I get:

Correct Image (8.9.2):
TIFF image data, little-endian, direntries=17, height=1595, bps=8, compression=JPEG, PhotometricIntepretation=BlackIsZero, orientation=upper-left, width=1595

And the new version creates (8.10.0):
TIFF image data, little-endian, direntries=16, height=1595, bps=1, compression=none, PhotometricIntepretation=BlackIsZero, orientation=upper-left, width=1595

In both cases the command was:
vips im_vips2tiff infile outfile:jpeg:90,tile:256x256,pyramid

Why has the bps reduced and compression now missing?

All help appreciated!

**Comment by jcupitt** [comment]
You're right, the new bps setting is broken in vips7 mode, I'll fix it.

As a workaround, I would switch to the new vips8 syntax. Try:

```
vips tiffsave gargoyle.jpg x.tif --compression jpeg --Q 90 --pyramid --tile --tile-width 256 --tile-height 256
```

It's wordier, but much easier to understand, and should be a bit faster. The old vips7 syntax is going to be dropped soon, so you'll need to update eventually.

You can also write:

```
vips copy gargoyle.jpg x.tif[compression=jpeg,Q=90,pyramid,tile,tile-width=256,tile-height=256]
```

Make sure there are no spaces.

**Comment by jcupitt** [comment]
This is fixed in 8.10 HEAD and should be in 8.10.1. Thanks!
